demo/utilities/router/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 	<head>
 		<title>Router Playground</title>
-		<script type="module" src="/example/app.js"></script>
+		<script type="module" src="/demo/utilities/router/app.js"></script>
```

Comment:
The router demo was broken. It now "works", but is still not great because if you navigate away and press the back button it's completely lost. This is due to how we have the repo setup and how the router inherently works - we'll need to come up with something else here.

Reply:
The router's going to move out of here into its own repo soonish anyway.